### PR TITLE
Release doc 2

### PR DESCRIPTION
**Pull request by watfagithub**
Resolves #

## Proposed Changes

-

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: pages/guide/09-Release-Notes.md [hunk 1]
```diff
@@ -5,6 +5,37 @@ title: Release Notes
 toc: true
 ---
 
+# Release [2.0.0-beta.1](https://github.com/yavin-dev/app/releases/tag/v1.0.0-beta.1)
```
Comment (kevinhinterlong) [inline pages/guide/09-Release-Notes.md]:
```suggestion
# Release [1.0.0-beta.2](https://github.com/yavin-dev/app/releases/tag/v1.0.0-beta.2)
```

File: pages/guide/09-Release-Notes.md [hunk 2]
```diff
@@ -5,6 +5,37 @@ title: Release Notes
...
+- [yavin-dev/framework#1334](https://github.com/yavin-dev/framework/pull/1334) Add week, isoweek, month grains for elide.
+- [yavin-dev/framework#1335](https://github.com/yavin-dev/framework/pull/1335) Refactor column-selector.
+
+<img src="/assets/images/Release_2_1335.png" width="200px">
+
+- [yavin-dev/framework#1296](https://github.com/yavin-dev/framework/pull/1296) Use chips for parameters in filter builder display.
+
+<img src="/assets/images/Release_2_1296.png">
+
+- [yavin-dev/framework#1295](https://github.com/yavin-dev/framework/pull/1295) Use chips for parameters in filter builder display.
```
Comment (kevinhinterlong) [inline pages/guide/09-Release-Notes.md]:
```suggestion
- [yavin-dev/framework#1295](https://github.com/yavin-dev/framework/pull/1295) Allow parameters in elide filters (e.g. filtering on Year grain, but requesting Day grain)
```

File: pages/guide/09-Release-Notes.md [hunk 3]
```diff
@@ -5,6 +5,37 @@ title: Release Notes
 toc: true
 ---
 
+# Release [2.0.0-beta.1](https://github.com/yavin-dev/app/releases/tag/v1.0.0-beta.1)
+{:.beta}
+
+### <i class="d-icon d-activity"></i> Updates & Additions
+
+- [yavin-dev/framework#1333](https://github.com/yavin-dev/framework/pull/1333) Use Ember concurrency for data interfaces/adapters.
```
Comment (kevinhinterlong) [inline pages/guide/09-Release-Notes.md]:
```suggestion
- [yavin-dev/framework#1333](https://github.com/yavin-dev/framework/pull/1333) Use Ember concurrency for data interfaces/adapters. (Enables cancelling typeahead queries)
  > This introduced a bug in the dimension selector where it expects the elide `pageSize` in your `application.yaml` to be
  > larger that the size of `<= SMALL` cardinality dimensions.
  > ```
  > elide:
  >   pageSize: 1234
  > ```
  >
  > If this is not set, some dimensions may appear to be missing from the dropdown.
  {:.warning}
```

File: pages/guide/09-Release-Notes.md [hunk 4]
```diff
@@ -5,6 +5,37 @@ title: Release Notes
...
+- [yavin-dev/framework#1295](https://github.com/yavin-dev/framework/pull/1295) Use chips for parameters in filter builder display.
+
+
+### Upgrade Steps
+
+You can upgrade your Yavin app version to `2.0.0-beta.1` with the following commands:
+  ```shell
+  cd app
+  rm -rf ui/node_modules
+  git pull origin
```
Comment (kevinhinterlong) [inline pages/guide/09-Release-Notes.md]:
```suggestion
  git pull origin
  ./gradlew clean
```

File: pages/guide/09-Release-Notes.md [hunk 5]
```diff
@@ -5,6 +5,37 @@ title: Release Notes
...
+- [yavin-dev/framework#1296](https://github.com/yavin-dev/framework/pull/1296) Use chips for parameters in filter builder display.
+
+<img src="/assets/images/Release_2_1296.png">
+
+- [yavin-dev/framework#1295](https://github.com/yavin-dev/framework/pull/1295) Use chips for parameters in filter builder display.
+
+
+### Upgrade Steps
+
+You can upgrade your Yavin app version to `2.0.0-beta.1` with the following commands:
```
Comment (kevinhinterlong) [inline pages/guide/09-Release-Notes.md]:
```suggestion
You can upgrade your Yavin app version to `1.0.0-beta.2` with the following commands:
```

**Review by jkusa — DISMISSED** [review]

**Comment by kevinhinterlong** [comment]
The version is listed as `2.0.0-beta.2`, should be `1.0.0-beta.2`

File: pages/guide/09-Release-Notes.md [hunk 6]
```diff
@@ -5,6 +5,45 @@ title: Release Notes
 toc: true
 ---
 
+# Release [2.0.0-beta.2](https://github.com/yavin-dev/app/releases/tag/v1.0.0-beta.2)
```
Comment (jkusa) [inline pages/guide/09-Release-Notes.md]:
```suggestion
# Release [1.0.0-beta.2](https://github.com/yavin-dev/app/releases/tag/v1.0.0-beta.2)
```

File: pages/guide/09-Release-Notes.md [hunk 7]
```diff
@@ -5,6 +5,45 @@ title: Release Notes
...
+- [yavin-dev/framework#1296](https://github.com/yavin-dev/framework/pull/1296) Use chips for parameters in filter builder display.
+
+<img src="/assets/images/Release_2_1296.png">
+
+- [yavin-dev/framework#1295](https://github.com/yavin-dev/framework/pull/1295) Allow parameters in elide filters (e.g. filtering on Year grain, but requesting Day grain)
+
+
+### Upgrade Steps
+
+You can upgrade your Yavin app version to `2.0.0-beta.2` with the following commands:
```
Comment (kevinhinterlong) [inline pages/guide/09-Release-Notes.md]:
```suggestion
You can upgrade your Yavin app version to `1.0.0-beta.2` with the following commands:
```

**Review by kevinhinterlong — APPROVED** [review]

**Pull request merged**